ui/tests/acceptance/pageobjects/helloPage.js

```diff
@@ -0,0 +1,21 @@
+const {join} = require(`path`)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['join' is assigned a value but never used.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)